flink-table/flink-sql-parser/src/main/codegen/data/Parser.tdd

```diff
@@ -394,23 +389,13 @@
   ]
 
   # List of methods for parsing ddl supported data types.
-  # Return type of method implementation should be "SqlIdentifier".
-  # Example: SqlParseTimeStampZ().
-  flinkDataTypeParserMethods: [
-    "SqlArrayType()",
-    "SqlMultisetType()",
-    "SqlMapType()",
-    "SqlRowType()",
-    "SqlStringType()",
-    "SqlBytesType()",
-    "SqlTimestampType()",
-    "SqlTimeType()"
-  ]
-
-  # List of methods for parsing custom data types.
-  # Return type of method implementation should be "SqlIdentifier".
+  # Return type of method implementation should be "SqlTypeNameSpec".
   # Example: SqlParseTimeStampZ().
   dataTypeParserMethods: [
+    "ExtendedSqlBasicTypeName()",
+    "CustomizedCollectionsTypeName()",
```

Comment:
change this name to `ExtendedSqlCollectionsTypeName` to be more align with other methods?

Reply:
We already have a `ExtendedSqlCollectionsTypeName` parse block name, i name it `CustomizedXXX` because the syntax is totally customized and does not belong to standard SQL.